device/mellanox/x86_64-mlnx_msn4700-r0/Mellanox-SN4700-C128/sai.profile

```diff
@@ -0,0 +1,4 @@
+SAI_VXLAN_SRCPORT_RANGE_ENABLE=1
```

Comment:
Why do we need SAI_VXLAN_SRCPORT_RANGE_ENABLE? I believe this is removed in latest branches. Please check